hw/ip/alert_handler/dv/Makefile

```diff
@@ -33,6 +33,9 @@ UVM_TEST_SEQ    ?= alert_handler_base_vseq
    UVM_TEST_SEQ   = alert_handler_sanity_vseq
  endif
 
+ ifeq (${TEST_NAME},alert_handler_random_alerts)
```

Comment:
Also add it to the hjson?

Reply:
Do you mean the testplan? This test is already included in the testplan :)

Reply:
No, the alert_handler_sim_cfg.hjson.

Reply:
Oh that is right! thanks for the reminder!